DOL.WHD.Section14c.Web/webpack.config.js

```diff
@@ -0,0 +1,65 @@
+var webpack = require("webpack");
+var path = require('path');
+
+module.exports = {
+    devtool: 'eval-source-map',
+	entry: './src/modules/app.js',
+	output: {
+		path: './',
+		filename: 'index.js'
+	},
+	//externals: {
+    //},
+    eslint: {
+        failOnWarning: false,
+        failOnError: false
```

Comment:
For the next couple of sprints, this is fine, while lots of new code is being added.  But eventually I'd like to see the build fail for lint errors to make it harder to ignore them.